cg/meta/qc_metrics/collect_qc_metrics.py

```diff
@@ -0,0 +1,85 @@
+"""Module for the collect qc metrics meta api."""
+
+import logging
+from typing import Iterator
+
+from housekeeper.store.models import File, Tag
+
+from cg.clients.arnold.api import ArnoldAPIClient
+from cg.clients.janus.api import JanusAPIClient
+from cg.clients.janus.dto.create_qc_metrics_request import (
+    CreateQCMetricsRequest,
+    FilePathAndTag,
+)
+from cg.clients.janus.exceptions import JanusClientError, JanusServerError
+from cg.exc import HousekeeperFileMissingError
+from cg.meta.meta import MetaAPI
+from cg.models.cg_config import CGConfig
+from cg.store.models import Case, Sample
+
+LOG = logging.getLogger(__name__)
+
+
+class CollectQCMetricsAPI(MetaAPI):
+    """Collect qc metrics API."""
+
+    def __init__(self, config: CGConfig):
+        super().__init__(config)
+        self.janus_api: JanusAPIClient = config.janus_api
+        self.arnold_api: ArnoldAPIClient = config.arnold_api
+
+    def get_qc_metrics_file_paths_for_case(self, case_id: str) -> list[File]:
+        """Get the multiqc metrics files for a case."""
+        files: list[File] = self.housekeeper_api.get_files(
+            bundle=case_id, tags=["qc-metrics", "janus"]
+        ).all()
+        if not files:
+            raise HousekeeperFileMissingError(f"No qc metrics files found for {case_id}.")
+        return files
+
+    @staticmethod
+    def get_tag_to_include(tags: list[Tag]) -> str:
+        """Get the multiqc module file tag."""
+        tags_to_exclude: list[str] = ["qc-metrics", "janus", "multiqc"]
```

Comment:
```suggestion
        tags_to_exclude: list[str] = ["qc-metrics", "janus", "multiqc", "storage"]
```

Reply:
And I assume you only want to send just one tag. What will happen if we need to add a pipeline specific tag to these multiqc files? Then, would we need to add it to this list? I'm a bit unsure why we don't decouple the tags from these files and leave this logic only in CG/hermes.

Reply:
fair consideration, did not think about that. Uff tough one to solve. Will think about it

Reply:
It's unlikely that we'll want to use these files for something other than sending them to Janus, but I guess it's something to take into account 🤔